### ISSUE TITLE
Docker and Gurobi

### ISSUE THREAD

**Issue by pat676** [verivital/vnn-comp#6]
Hi, 

Our toolkit depends on Gurobi and it seems like Gurobi has blocked Docker with the standard academic license. The output is:

"gurobipy.GurobiError: License is not valid in a Docker container, contact Gurobi support for more options” 

I don’t see any way around this, and I believe several toolkits use Gurobi. 

We could contact Gurobi to find a solution as the error message suggest; however, I don’t believe this is realistic within the current timeframe. My suggestion is to remove the requirement for docker during this run and find a solution for next years competition.

**Comment by ttj** [comment]
> We could contact Gurobi to find a solution as the error message suggest; however, I don’t believe this is realistic within the current timeframe. My suggestion is to remove the requirement for docker during this run and find a solution for next years competition.

I agree, this is good to know for the future, so thanks for looking into it in more depth, and sorry it ended up being more complicated. Please just provide (1) a script that will set up your tool/dependencies (on a standard setup, eg, recent Ubuntu version) and (2) a script to reproduce/run everything. We can then look into this further for future iterations.
